src/www/ui/api/helper/DbHelper.php

```diff
@@ -0,0 +1,157 @@
+<?php
```

Comment:
please consider file headers for all new files:
https://github.com/fossology/fossology/wiki/Coding-Style#default-license-and-file-headers